### PR TITLE
Added the blog related to issue-881

### LINKED ISSUE #881: [Add] <Wish to add new Blog in the Blog Section>
### Site URL

https://girlswhocode.com/

### Description of the above provided site

Girls Who Code is an international nonprofit organization that strives to close the gender gap in technology . It aims to support and increase the participation of women in computer science and technology . 

### Category

Resources

### Sub Category

Blogs

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by SargamPuram**
PR fixes an open issue , Closes #881 . 
I have added the blog in the Blogs subcategory within the resources category . 
Kindly review this .
Screenshot
LH2(2)

### CODE REVIEW

File: database/resources/blogs.json [hunk 1]
```diff
@@ -117,5 +117,12 @@
         "url": "https://blog.openreplay.com/",
         "category": "resources",
         "subcategory": "blogs"
+    },
+    {
+        "name": "Girls Who Code",
+        "description": "Girls Who Code is an international nonprofit organization that strives to close the gender gap in technology . It aims to support and increase the participation of women in computer science and technology.",
+        "url": "https://girlswhocode.com/",
```
Comment (CBID2) [inline database/resources/blogs.json:124]: [Proofreading]
```suggestion
        "url": "https://girlswhocode.medium.com/",
```

Comment (SargamPuram) [reply database/resources/blogs.json]: sorry for the error , I won't happen again . I have updated it

**Comment by SargamPuram** [comment]
@CBID2 sorry for the error . I assure you that it won't happen again . I have updated the url  , kindly review it .

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Review by ujjawaltyagii — DISMISSED** [review]
Lgtm !! 👍  (please resolve the conversation)

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**